share/patch/patch_db_zonemaster_backend_ver_11.1.0.pl

```diff
@@ -157,16 +159,118 @@ sub patch_db_mysql {
         _update_data_result_entries( $dbh, 50000 );
 
         print( "\n-> (2/2) Normalizing domain names\n" );
-        _update_data_nomalize_domains( $db );
+        _update_data_normalize_domains( $db );
 
         $dbh->commit();
     } catch {
-        print( "Could not upgrade database:  " . $_ );
+        print( "\nCould not upgrade database:  " . $_ );
 
         $dbh->rollback();
     };
 }
 
+sub _patch_db_postgresql_step1 {
+    my ($dbh, $chunk_size) = @_;
```

Comment:
When called, only one parameter is included in the call. What is the purpose of the second parameter?

Reply:
The script alternates between fetching rows from the SELECT query and inserting them into the table by means of COPY, thus splitting one huge batch in many smaller batches. The `$chunk_size` parameter, by default set to 100 000, sets the number of rows fetched and inserted in each of those smaller batches.

Reply:
That is probably fine, but I think it is funny to have an input parameter to the `_patch_db_postgresql_step1()` that is never used in the call. It is a detail.